app/views/admin/admin_pages/index.html.erb

```diff
@@ -37,45 +41,29 @@
           <i class="fa-solid <%= admin_icon(user) %>"></i>
         </td>
         <td>
-          <span class="badge badge-pill badge-<%="warning" if user.type_of == "gold" %>"><%= user.type_of %></span>
+          <span class="badge rounded-pill bg-<%= user.gold? ? "warning" : "secondary" %>"><%= user.type_of %></span>
           <!-- Button trigger modal -->
           <button type="button"
-                  class="btn <%= type_of_btn(user) %> <%= disabled_btn(user) %>"
-                  data-toggle="modal"
-                  data-target=<%= "#confirmModal" + user.id.to_s %>>
+                  class="<%= type_of_btn(user) %> <%= disabled_btn(user) %>"
+                  data-bs-toggle="modal"
+                  data-bs-target=<%= "#confirmModal" + user.id.to_s %>>
             <%= act(user) %>
           </button>
-        </td>
-        <td>
-          <button class="btn btn-info btn-block" data-toggle="modal" data-target= <%= "#followersModal" + user.id.to_s %>>
-            <i class="fa-solid fa-address-book"></i>
-          </button>
-        </td>
-        <td>
-          <button class="btn btn-info btn-block" data-toggle="modal" data-target= <%= "#followingModal" + user.id.to_s %>>
-            <i class="fa-regular fa-address-book"></i>
-          </button>
+          <%= render "upgrade_confirm", user: user %>
         </td>
         <td>
           <div class="btn-group btn-block" role="group" aria-label="User actions">
-            <button class="btn btn-info" data-toggle="modal" data-target= <%= "#userModal" + user.id.to_s %>>
-              <i class="fa-solid fa-info"></i>
-            </button>
-            <button class="btn btn-danger"><i class="fa-solid fa-user-slash"></i></button>
+            <%= render "shared/user_info_modal", user: user %>
+            <%= button_to "Delete", user, method: :delete, data: { confirm: "You sure?" }, class: "link-danger bg-light" %>
```

Comment:
confirm: "You sure?" i18n đc k bro

Reply:
sửa được mà tôi quên mất. tôi sửa lại rồi nhé.